ios/FluentUI/Pill Button Bar/PillButtonStyle.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+// MARK: - PillButtonStyle
+
+@available(*, deprecated, renamed: "PillButtonStyle")
+public typealias MSPillButtonStyle = PillButtonStyle
+
+@objc(MSFPillButtonStyle)
+public enum PillButtonStyle: Int {
```

Comment:
Not in the scope of this PR (and we also discussed about it offline), this change breaks the public API and the next release version ideally should reflect that (bumping at least the minor version).
@harrieshin we can discuss this further to make our release numbers more consistent with the rules of Semantic Versioning.